### DISCUSSION TITLE
Renault kangoo radio not configured

### DISCUSSION THREAD

**Discussion by Andy025265** [cedricp/ddt4all#1133] [Q&A]
![f7595fc34777f0cae90e57a1acbc8ea7](https://github.com/cedricp/ddt4all/assets/171626605/c4f2200c-b398-4342-ada1-abcb5736b3ec)
![IMG_0230](https://github.com/cedricp/ddt4all/assets/171626605/985c37c2-6f56-4d3b-8cb5-68a60514b3d9)

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/discussions/1055

**Reply by Andy025265** [reply]
What’s that code mean

**Reply by Furtif** [reply]
Is not a code click in is a links

**Reply by Andy025265** [reply]
Thanks

**Reply by Furtif** [reply]
fine, look and read also https://github.com/cedricp/ddt4all/discussions/970
